### ISSUE TITLE
"ReferenceError: CMS is not defined" when loading a page with the CMS toolbar after upgrading to django-cms 3.5.0

### ISSUE THREAD

**Issue by timgraham** [divio/django-cms#6270]
Errors appear in the JavaScript console when loading a page with the CMS toolbar (add ?edit to the URL):

ReferenceError: CMS is not defined bundle.toolbar.min.js:1:389068
ReferenceError: CMS is not defined localhost:8000:10:1   

Sample HTML:
```

<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8" />
    <title>Home</title>
    
<script data-cms src="/static/cms/js/dist/3.5.0/bundle.toolbar.min.js" type="text/javascript"></script>
<script data-cms>
CMS._plugins = CMS._plugins || [];
```
I bisected the change to "Introduced Webpack 3.0 " 1028cf2192856540f9594787f6d8715a914baccf.

### Environment

* Python version: 3.5
* Django version: 1.11.10
* django CMS version: 3.5.0

**Comment by czpython** [comment]
Thanks @timgraham.
Does this happen on all pages?

**Comment by timgraham** [comment]
Yes

**Comment by vxsx** [comment]
Looks like CMS now expects the JS to be run only after the toolbar, so short term solution would be to move the `{% render_block "js" %}` after `{% cms_toolbar %}`.

Will be fixed in 3.5.1, sorry about that.
